Sources/PenguinParallel/NonblockingThreadPool/NonBlockingThreadPool.swift

```diff
@@ -64,7 +64,13 @@ public class NonBlockingThreadPool<Environment: ConcurrencyPlatform>: ComputeThr
   let totalThreadCount: Int
   let externalFastPathThreadCount: Int
   var externalFastPathThreadSeenCount: Int = 0
-  let coprimes: [Int]
+
+  /// An array of step-sizes that are each coprime with `queues.count`.
+  ///
+  /// When looking for work, pool threads will pick a random step size and traverse the `queues`
+  /// looking to steal work. The coprime property ensures that every queue will be examined, but the
+  /// stealing threads will traverse in diverging orders, avoiding thundering herds.
+  let stepSizes: [Int]
```

Comment:
```suggestion
  let workStealingStrides: [Int]
```

ersumpn'.  They're not just any collection of step sizes.